src/main/java/org/microg/nlp/backend/nominatim/BackendService.java

```diff
@@ -182,7 +210,7 @@ public void run() {
             synchronized (done) {
                 try {
                     Log.d(TAG, "Requesting " + url);
-                    HttpURLConnection connection = (HttpURLConnection) new URL(url)
+                    HttpsURLConnection connection = (HttpsURLConnection) new URL(url)
```

Comment:
> Casting this to a HttpsURLConnection instance does not improve security ;)

Yes obviously :)
But maybe it can be a good practice to use HttpsURLConnection object in case we need in the futur to play with the HTTPS connection parameters.